_posts/2016-08-25-build-opinions-to-become-wise.markdown

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title:  "Build opinions to become wise"
+date:   2016-08-25 12:08:27
+categories: [post]
+img: /img/myselfie.jpg-large
+tags: [me]
+summary: ""
+priority: 0.6
+changefreq: yearly
+---
+I am Gianluca, I am 24 years old and I work like Software Engineer.
+
+I like my work because there are really a lot of different kinds of Software
```

Comment:
I like my work because there are really a lot of different kinds on Software Engineering